### ISSUE TITLE
macOS "Get Info" dialog lists non-free license

### ISSUE THREAD

**Issue by sdaitzman** [VSCodium/vscodium#311]
**Describe the bug**

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.
  - This does happen in Microsoft's build but it's correct in their case

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Go to Applications folder
2. Right click on VSCodium
3. Click "Get Info"

**Expected behavior**
The Copyright field should reference the MIT licensing.

**Actual Behavior**
The Copyright field references a Microsoft all-rights-reserved copyright.

**Screenshots**
<img width="377" alt="Screen Shot 2020-01-01 at 9 37 24 AM" src="https://user-images.githubusercontent.com/1499476/71642508-52806580-2c7a-11ea-82e2-cfeda08087c2.png">

<img width="377" alt="Screen Shot 2020-01-01 at 9 37 39 AM" src="https://user-images.githubusercontent.com/1499476/71642514-6cba4380-2c7a-11ea-93eb-1e9a770c576c.png">

**Desktop (please complete the following information):**
 - OS: macOS 10.15.2
 - Architecture x64
 - Version 1.41.1

**Comment by stripedpajamas** [comment]
As far as I can tell, this is sourced from https://github.com/microsoft/vscode/blob/master/build/lib/electron.ts#L44

Should be easy enough to replace this pre-build with the proper text.

**Comment by stripedpajamas** [comment]
I am not a lawyer; I'm wondering what the proper text in this field should be? Do we have any examples of MIT-licensed binaries we can look at?

I checked a couple of open source apps I have installed (Brave, Gimp, DBeaver, Patchwork, RocketChat, Signal) and they all seem to be `[Copyright] (C) <year> <company|individual name> [All rights reserved]` (where [] denotes "some have" and <> denotes "all have").
